framework/auth/views.py

```diff
@@ -130,17 +132,18 @@ def auth_login(auth, **kwargs):
     if status_message == 'expired':
         status.push_status_message('The private link you used is expired.')
 
-    code = http.OK
     if next_url:
         status.push_status_message(language.MUST_LOGIN)
-        # Don't raise error if user is being logged out
-        if not request.args.get('logout'):
```

Comment:
I inherited this code with the L142-43 removals already made... which meant these lines did nothing (`code` is unused). So I'm assuming the developer before me made good assumptions : ). 

My manual tests confirm that at least no unexpected exceptions are raised.